.eslintignore

```diff
@@ -4,4 +4,5 @@ node_modules
 dist
 .env
 .envrc
-src/apollo/types.ts
\ No newline at end of file
+src/apollo/types.ts
+src/apollo/apollo-helpers.ts
```

Comment:
You added 15k lines of types and not used a single one of them.

Reply:
`apollo-helpers` are from this commit: https://github.com/mirumee/saleor-sdk/commit/dfe500e1dcc6c00db11cc0b421c8cad1a3a96c6a . I added only rule to eslintignore